### DISCUSSION TITLE
UI: Support order&limit to table graph when showing metrics

### DISCUSSION THREAD

**Discussion by CzyerChen** [apache/skywalking#12057] [Ideas]
Is it possible to support order & limit to table graph when showing metrics?

order: asc/des, means order by count desc/asc
limit: number, means shown topn

Example: show top10 order by count desc
<img width="1435" alt="image" src="https://github.com/apache/skywalking/assets/22817918/e39e9776-6208-47cc-95ee-d99a5281cd82">

**Comment by wu-sheng** [comment]
That should be implemented in MQE. I think.

What is your use case?

**Reply by CzyerChen** [reply]
List aggregate labeled values in table, and I want to order them and limit the shown row count.

**Reply by wu-sheng** [reply]
All logic should be implemented in MQE. Note, https://github.com/apache/skywalking/issues/11992 is on going, so MQE codes may have some changes about labels.
